src/vos/evtree.c

```diff
@@ -2625,22 +2608,23 @@ evt_common_insert(struct evt_context *tcx, struct evt_node *nd,
 
 	/* NB: can use binary search to optimize */
 	for (i = 0; i < nd->tn_nr; i++) {
-		struct evt_rect_df	df_tmp;
-		int			nr;
+		struct evt_rect	rtmp;
+		int		nr;
 
-		ne = evt_node_entry_at(tcx, nd, i);
+		evt_node_rect_read_at(tcx, nd, i, &rtmp);
 
-		evt_rect_write(&df_tmp, &ent->ei_rect);
-		rc = cb(tcx, nd, &ne->ne_rect, &df_tmp);
+		rc = cb(tcx, nd, &rtmp, &ent->ei_rect);
 		if (rc < 0)
 			continue;
 
 		if (!leaf) {
 			nr = nd->tn_nr - i;
-			memmove(ne + 1, ne, nr * sizeof(*ne));
+			memmove(&nd->tn_child[i + 1], &nd->tn_child[i],
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?